garak/evaluators/base.py

```diff
@@ -72,12 +73,19 @@ def evaluate(self, attempts: Iterable[garak.attempt.Attempt]) -> None:
                             hitlog_mode = (
                                 "w" if _config.transient.hitlogfile is None else "a"
                             )
+                            # why is report prefix a condition of placing file in the report_dir?
```

Comment:
I don't think it is here - the logic is just building a filename. I suspect that with the updated config changes & this XDG PR, this logic can pruned out here, so that one relies solely on the configured values with no conditionals.